backend/btrixcloud/subs.py

```diff
@@ -378,14 +382,77 @@ async def get_billing_portal_url(
 
         return SubscriptionPortalUrlResponse()
 
+    async def get_execution_minutes_price(self, org: Organization):
+        """Fetch price for addon execution minutes from external subscription app"""
+        if not org.subscription:
+            raise HTTPException(
+                status_code=404, detail="Organization has no subscription"
+            )
+        if external_subs_app_api_url:
+            try:
+                async with aiohttp.ClientSession() as session:
+                    async with session.request(
+                        "GET",
+                        f"{external_subs_app_api_url}/prices/additionalMinutes",
+                        headers={
+                            "Authorization": "bearer " + external_subs_app_api_key,
+                        },
+                        raise_for_status=True,
+                    ) as resp:
+                        json = await resp.json()
+                        return AddonMinutesPricing(**json)
+            # pylint: disable=broad-exception-caught
+            except Exception as exc:
+                print("Error fetching execution minutes price", exc)
```

Comment:
Same comment as below, (re)raising seems like the right move here

Reply:
This one I think maybe not, it should fail silently here, the frontend handles empty values for self-hosted installations.